src/domain/enrolment/__tests__/CreateEnrolmentPage.test.tsx

```diff
@@ -398,66 +398,74 @@ test('render and focuses error notification correctly', async () => {
   expect(screen.getByRole('textbox', { name: 'Nimi' })).toHaveFocus();
 });
 
-test('amount of adults is limited to 1 when there is 1 seat left after children', async () => {
-  render(<CreateEnrolmentPage />, {
-    mocks: mock3,
-    query: { eventId: eventId, occurrences: occurrenceIds },
-  });
-  await waitFor(() => {
-    expect(screen.getByLabelText(/lapsia/i)).toBeInTheDocument();
-  });
-  userEvent.type(screen.getByLabelText(/lapsia/i), '19');
-  userEvent.type(screen.getByLabelText(/aikuisia/i), '2');
-  userEvent.tab();
-  await waitFor(() => {
-    expect(
-      screen.getByText(
-        /Arvon tulee olla enintään 1 yhdessä lasten lukumäärän kanssa/i
-      )
-    ).toBeInTheDocument();
+describe('max group size validation of the Children and Adults -fields', () => {
+  const createEnrolmentForm = async (
+    childrenCount: string,
+    adultsCount: string
+  ) => {
+    render(<CreateEnrolmentPage />, {
+      mocks: mock3,
+      query: { eventId: eventId, occurrences: occurrenceIds },
+    });
+    await waitFor(() => {
+      expect(screen.getByLabelText(/lapsia/i)).toBeInTheDocument();
+    });
+    userEvent.type(screen.getByLabelText(/lapsia/i), childrenCount);
+    userEvent.type(screen.getByLabelText(/aikuisia/i), adultsCount);
+    userEvent.tab();
+  };
+
+  test('both of the fields are greater than the max group size', async () => {
+    createEnrolmentForm('21', '22');
+    await waitFor(() => {
+      expect(
+        screen.getAllByText(
+          /Lasten ja aikuisten yhteislukumäärän tulee olla enintään 20/i
+        )
+      ).toHaveLength(2);
+    });
   });
-});
 
-test('amount of adults is limited to 0 when children fills the group in whole', async () => {
-  render(<CreateEnrolmentPage />, {
-    mocks: mock3,
-    query: { eventId: eventId, occurrences: occurrenceIds },
-  });
-  await waitFor(() => {
-    expect(screen.getByLabelText(/lapsia/i)).toBeInTheDocument();
+  test('one field is greater than the max group size and another one is (still) empty', async () => {
+    createEnrolmentForm('21', '');
```

Comment:
createEnrolment calls are missing await and I'm getting this error:

<img width="1041" alt="Screenshot 2021-02-19 at 14 42 04" src="https://user-images.githubusercontent.com/15219142/108506469-81fc9200-72c1-11eb-9bcc-a97a6621ff14.png">

Reply:
True that! Strange that I didn't get this error highlighted. However, I think it's now fixed.